dist/tools/pioasm/Makefile

```
@@ -0,0 +1,16 @@
+PKG_NAME=pioasm
+PKG_URL=https://github.com/raspberrypi/pico-sdk.git
+# release SDK 1.5.0
+PKG_VERSION=2ccab115de0d42d31d6611cca19ef0cd0d2ccaa7
+PKG_LICENSE=BSD-3-Clause
+PKG_PATCH_DIR=$(RIOTTOOLS)/pioasm/patches
+PKG_BUILD_DIR=$(PKG_SOURCE_DIR)/tools/pioasm
+
+include $(RIOTBASE)/pkg/pkg.mk
+
+all: $(CURDIR)/pioasm
+
+$(CURDIR)/pioasm:
```

```suggestion
$(CURDIR)/pioasm:  $(PKG_DOWNLOADED)
```

I absolutely don't know if it's a dirty hack or not. But it fixes the issue locally on my side !

---

>  worked 3 times in a row with -j 5

I forgot to delete the `pioasm` binary ...
Thank you for the fix. I even had to add `PKG_PREPARED` as a prerequisite.
Else it complained that there was no RIOT output format.
You already see this in the code base in combination with `cmake`.
When I deleted `elf2uf2` I got the same problems, so I fixed it in another commit.
And I moved `tests/periph_pio` to `tests/periph/pio`.